examples/src/main/java/com/mapbox/navigation/examples/core/MultipleStopsActivity.kt

```diff
@@ -188,6 +204,7 @@ class MultipleStopsActivity : AppCompatActivity(), OnMapReadyCallback {
         super.onDestroy()
         mapboxNavigation?.stopTripSession()
         mapboxNavigation?.onDestroy()
+        mapboxNavigation?.detachArrivalObserver()
```

Comment:
want to find a way to be able to remove this

Reply:
changed this to be functional. removeArrivalController will disable the feature and allow devs to do things manually.